docs/UserGuide.adoc

```diff
@@ -120,14 +127,19 @@ Format: `(ab)edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [b/BIRTH
 ****
 * Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list. The index *must be a positive integer* 1, 2, 3, ...
 * At least one of the optional fields must be provided.
-* If a field is specified but the parameter is left empty, the entire field will be deleted.
-* Existing values will be updated to the input values except tags.
+* If a field is specified but the parameter is left empty, the entire field will be deleted (except for the `TAG_TO_BE_ADDED`, `NAME` and `PHONE_NUMBER` prefixes).
+* Existing values will be updated to the input values (except tags).
+* Tags will be appended or deleted according to the prefix specified.
 * Remarks cannot be edited using this command.
 ****
 
 [TIP]
-* Multiple tags can be specified and all will be added, if it is not a duplicate.
-* Multiple tags can be specified and all will be deleted, if they exist.
+Multiple tags can be specified and all will be added, if it is not a duplicate.
```

Comment:
maybe this can say sth like "Multiple to-be-added tags can be specified..." so that it's clearer. Same for the next one

Reply:
ok!